### PR TITLE
Use --enable-build-raft for dqlite package

### PR DESCRIPTION
**Pull request by cole-miller**
Signed-off-by: Cole Miller <cole.miller@canonical.com>

### CODE REVIEW

File: debian/control [hunk 1]
```diff
@@ -4,7 +4,6 @@ Priority: optional
 Maintainer: Mathieu Borderé <mathieu.bordere@canonical.com>
```
Comment (letFunny) [inline debian/control]: Should we change this?

**Review by letFunny — APPROVED** [review]
I am a bit uneasy seeing there is no CI but I guess this will be in Launchpad, right?

File: debian/control (hunk 1 above)
Comment (cole-miller) [reply debian/control]: Done

**Comment by cole-miller** [comment]
> I am a bit uneasy seeing there is no CI but I guess this will be in Launchpad, right?

Yes but also you're right, testing Launchpad builds/uploads is something I've wanted to do for a while. I added a CI job to test that we can at least successfully upload to the canary PPA (not the normal one obviously), let's see if it works as intended...

**Comment by cole-miller** [comment]
I've confirmed manually using the "canary" PPA that the latest revision here can build dqlite binary packages ([example build](https://launchpad.net/~dqlite/+archive/ubuntu/canary/+build/28904659)). I did experiment with setting up CI to do this (would be useful for the dqlite repo as well to avoid accidentally merging changes that don't build on armhf), but I don't want to expose the dqlitebot secret key (which can also upload to the real PPA) to arbitrary pull requests, so I think we'll need to create a separate user who's not on the dqlite team in order to do this. In the meantime I'm going to merge this.

**Pull request merged**